alvr/common/src/commands.rs

```diff
@@ -21,6 +21,10 @@ pub fn exec_fname(name: &str) -> String {
 pub fn exec_fname(name: &str) -> String {
     format!("{}.exe", name)
 }
+#[cfg(target_os = "macos")]
+pub fn exec_fname(name: &str) -> String {
+    format!("{}.exe", name)
+}
```

Comment:
You can remove this and instead change the Linux one to be `target_family = "unix"`

Reply:
The function body is a mistake. I'll fix it